index.bs

```diff
@@ -313,6 +313,35 @@ See also
 
 * [[DESIGN-PRINCIPLES#do-not-expose-use-of-assistive-tech]]
 
+<h3 class=question id=hidden-data>
+  Does data exposed by your specification carry related but distinct
+  information that may not be obvious to users?
+</h3>
+
+Features which enable users
+to share data with origins
+should ensure that such data
+does not carry related but distinct metadata
+without the user's understanding and consent.
+
+Documents
+such as image or video files
+often contain metadata about
+where and when the image, video, or audio was captured
+and
+what kind of device captured or produced the data.
+When uploaded,
+this kind of metadata
+may reveal to origins
+information the user did not intend to reveal,
+such as the user's location
+and socioeconomic status.
```

Comment:
Not sure that socioeconomic status is the top worry ("an expensive camera was used here"?). But in the presence of systems that do object, and face detection; and object, location, and face recognition, it's not a stretch to realize that the metadata includes such analytic information.